src/content/post/20161129_git_reflog_amend.md

```diff
@@ -11,104 +11,103 @@ author      = "Marcello"
-That's OK, because within Git most of the time lies the appropriate and easiest solution. So, let me tell you how I was hit by an `--amend` during a commit and how I easily got out.
+No need to worry though, since Git is equally able to put you in trouble and save your day with the same ease. So, let me tell you how I was hit by an `--amend` during a commit and how I easily got out.
 
 <!--more-->
 
-Sometimes during the day, especially in the time right before lunch or going home, it's possible to forget one the most important rules of the savvy developer: in the terminal, you can be quick with all the keys, but think twice before pressing "enter".
+Sometimes during the day, especially in the time right before lunch or leaving work, it's possible to forget one the most important rules of the savvy developer: in the console, you can be quick with all the keys, but think twice before pressing "enter".
 
-Breaking this rule is what happened to me some days ago, but that was luckily a chance to remind me how much powerful git is and how sometimes the solutions to an apparently bad situation can be just a few commands away.
+Sure, I managed to break that very rule some days ago. Luckily that was a chance to remind me how much powerful Git is and how sometimes the solutions to an apparently bad situation can be just a few commands away.
 
 ## The amend secret
 
 As many of you probably know, the `git commit --amend` command can be used to edit the message of the last commit in the history. This is one of the most common uses because - as developers - we are prone to typos and bad phrasing.
 But the `--amend` option can also be used to include *new code* in the last commit.
 
-For example.
+For example:
 Let's say you've just created a commit but you've also forgot to add a block comment to that very method you've written, and you don't want to have another commit in the history, for various reasons. You have many choices at this point, including creating a new commit with the new lines and squash it with the previous one using the `git rebase` command.
 
 But you can also do this:
 
-<pre>
+``` bash
 # Add new line and prepare them for the commit.
 git add -p
 # Amend the previous commit and add to it the staged modifications.
 git commit --amend
-</pre>
+```
 
 A this point, you are asked to edit the previous commit message and the staged edits will be included in it. Useful, isnt'it?
 
 ## The side effect
 
-This little trick can be very handy in many situations, but it can also be a very sharp double-edge blade because, as you probably have guessed, you can use the amend on *any* commit including, for example, a *merge commit*.
+This little trick can be very handy in many situations, but it can also be a very sharp double-edged blade because, as you probably have guessed, you can use the amend on *any* commit including, for example, a *merge commit*.
```

Comment:
You changed sword to blade but sword was right. The idiom for "lama a doppio taglio" is "double-edged sword" :)